packages/cli/src/commands/compile.ts

```diff
@@ -45,17 +46,19 @@ const command: GluegunCommand = {
       plugins.push(compileAwayBuilderComponents());
     }
 
+    const mitosisConfig = getMitosisConfig();
+    const generatorOptions = mitosisConfig?.options?.[to];
+
     const generatorOpts: Partial<{ [K in AllGeneratorOptionKeys]: any }> = {
+      ...generatorOptions,
       prettier: opts.prettier ?? true,
       plugins: plugins,
```

Comment:
One small detail I noticed here: we have a `plugins` array that is declared here in the function. We want to merge it with the contents of `generatorOptions.plugins` if there are any. I think that'll look like this:

```suggestion
    const mitosisConfig = getMitosisConfig();
    const generatorOptions = mitosisConfig?.options?.[to];

    const generatorOpts: Partial<{ [K in AllGeneratorOptionKeys]: any }> = {
      ...generatorOptions,
      prettier: opts.prettier ?? true,
      plugins: [
        ...plugins,
        ...generatorOptions.plugins
      ],
```